src/format/opamFile.ml

```diff
@@ -2903,6 +2903,8 @@ module Dot_installSyntax = struct
     share   : (basename optional * basename option) list;
     share_root: (basename optional * basename option) list;
     etc     : (basename optional * basename option) list;
+    etc_root: (basename optional * basename option) list;
```

Comment:
As you modify the file syntax, you need to update [`format_version`](https://github.com/ocaml/opam/pull/3958/files#diff-017f3d0ef2400d5d3562b364ea6a5db6R2895) to `2.1`